services/net/contentmigration/migrators/PaperMigrator.cs

```diff
@@ -115,6 +121,42 @@ public override SourceContent CreateSourceContent(LookupModel lookups, SourceMod
         return content;
     }
 
+    /// <summary>
+    /// fix the replace of paragraph markers with "|"
+    /// </summary>
+    /// <param name="text"></param>
+    /// <param name="summary"></param>
+    /// <returns>formatted body string</returns>
+    private string GetNewsItemBody(string text, string summary)
+    {
+        string body = string.Empty;
+
+        if (!string.IsNullOrEmpty(text))
+        {
+            string const marker = '|';
+            int index = text.IndexOf(marker);
+            if (index !== -1)
+            {
+                // found at least one linebreak marker
+                StringBuilder newBody = new StringBuilder();
+                newBody.Append("<p>");
+                newBody = text.Replace(marker, "</p><p>");
```

Comment:
Won't this result in an extract `<p></p>` every time?  Why can't we replace with `\n`?

Reply:
I think this is clearer `$"<p>{text.Replace(marker, "</p><p>")}</p>"`